docs/equality.md

```diff
@@ -37,26 +37,26 @@ This structure can be combined further:
 
 ```ts
 import { eq } from "fp-ts";
+import { Eq } from "fp-ts/lib/Eq";
 
 type Point = {
   x: number;
   y: number;
 };
 
-const eqPoint: eq.Eq<Point> = eq.getStructEq({
-  x: eq.eqNumber,
-  y: eq.eqNumber,
-});
+const eqPoint: Eq<Point> = {
+  equals: (p1: Point, p2: Point) => p1 === p2 || (p1.x === p2.x && p1.y === p2.y)
+}
 
 type Vector = {
   from: Point;
   to: Point;
 };
 
-const eqVector: eq.Eq<Vector> = eq.getStructEq({
-  from: eqPoint,
-  to: eqPoint,
-});
+const eqVector: Eq<Vector> = {
```

Comment:
We can use https://gcanti.github.io/fp-ts/modules/Eq.ts.html#struct here.

Reply:
Fixed it.